app/src/main/res/values/strings.xml

```diff
@@ -103,12 +103,15 @@
   <string name="onboarding_location_blurb">Allow the app your location and visit all our sponsors’ booths to win swag 🎁</string>
   <string name="onboarding_location_explanation">We’ll need to ask for access to the device location, and to have the Bluetooth turned on</string>
   <string name="onboarding_location_cta">✨ I like swag! ✨</string>
-  <string name="onboarding_error_permission_denied">We need the location permission to continue.</string>
-  <string name="onboarding_error_location_denied">We need the location permission to continue.</string>
-  <string name="onboarding_error_bluetooth_denied">We need the Bluetooth to be active to continue.</string>
-  <string name="onboarding_error_location_failed">Unable to verify location settings for the device.</string>
-  <string name="onboarding_error_location_failed_action">ENABLE MANUALLY</string>
-  <string name="onboarding_error_google_client_connection">Unable to verify location settings for the device. Please try again shortly.</string>
+
+  <string name="proximity_error_not_opted_in">Sorry, unable to opt in because… you’re not opted in? If that makes any sense.</string>
```

Comment:
Yeah but what I meant is "You're not opted in" isn't correct, the correct form is "You didn't opt in"